src/bp/ui-admin/src/Pages/Components/Licensing/UpdateLicenseModal.js

```diff
@@ -58,13 +63,24 @@ export default class UpdateLicenseModal extends React.Component {
           <strong>Current total:</strong> {license.cost || 0}$
         </span>
         <Row className="align-items-center">
-          <Col md="6">
+          <Col md="4">
             <strong>
               New total: <span className="text-brand">{(this.state.order && this.state.order.totalPrice) || 0}$</span>
             </strong>
           </Col>
-          <Col md="6" className="text-right">
-            <Button color="primary" onClick={this.updateLicense} disabled={!this.state.isDirty || this.state.isLoading}>
+          <Col md="4">
+            <Input type="checkbox" name="terms" onChange={this.toggleReadTerms} />
+            <Label check>
+              I accept the <a href="https://botpress.io/company/terms/">Terms of Service</a> and the{' '}
```

Comment:
use `&nbsp;` which is natively supported by the browser instead of `{' '}`

Reply:
this the auto-format that does a line wrap